tools/runner/client.go

```diff
@@ -48,6 +48,26 @@ func NewLoadTestGetter() clientset.LoadTestGetter {
 		return nil
 	})
 
+	schemebuilder.AddToScheme(clientgoscheme.Scheme)
+	scheme := clientgoscheme.Scheme
+	types := scheme.AllKnownTypes()
+	_ = types
+
+	return clientset.LoadTestV1().LoadTests(corev1.NamespaceDefault)
+}
+
+// NewGRPCTestClientSet returns a new GRPCTestClientset.
```

Comment:
Here I would separate NewGRPCTestClientset (note lowercase s) fom NewK8sClientset, with the second one returning a vanilla client set. I would import k8s.io/kubernetes/clientset as k8sclientset to achieve that.

Thinking about it more, perhaps the interfaces we want are

func NewLoadTestGetter(namespace corev1.Namespace) clientset.LoadTestGetter

We could leave the clientset New methods as they are or build them into the two functions that we really want to use.

and

func NewPodsGetter(namespace corev1.Namespace) corev1.PodsGetter

Reply:
I separated the clientset functions (and changed the s).

I went ahead and renamed it to NewGRPCTestClientset. It now gets the grpc clientset itself by calling NewGRPCTestClientset.

It looks to me like a namespace argument in NewPodsGetter would be unused, so I did not add this. Am I misunderstanding something?
For NewLoadTestGetter, the namespace is used but because (1) the namespace never changes and (2) this function is called from main and would add unnecessary imports/complexity, I did not add this argument. But again please let me know if I am confusing something.